elastic/kibana
# [Reporting] use data-test-subj on toast to check for failure

Depends on: https://github.com/elastic/eui/pull/1302
Resolves: https://github.com/elastic/kibana/issues/25418

## Summary

This fixes a regression bug in Reporting's detection of Error and Warning toast notifications. 

### Screenshot
![image](https://user-images.githubusercontent.com/908371/48292183-95f49080-e436-11e8-92d8-7cc3a0f5c6e0.png)


## To test:
1. Start a Kibana "A" instance:
   ```
   node scripts/kibana -c config/kibana.dev.xpack.yml
   ```
2. Start a Kibana "B" instance with reporting polling disabled:
   ```
   node scripts/kibana -c config/kibana.dev.xpack.yml --kibana.index=.kibana-boo --server.port=5661 --xpack.reporting.queue.pollEnabled=false
   ```
3. Create a dashboard in the "B" instance (http://localhost:5661)
4. Queue a report of the dashboard in the "B" instance
   - The "A" instance should claim the reporting job
5. Report should fail with a clear message

### Checklist
